src/backend/libpq/rangerrest.c

```diff
@@ -463,7 +464,12 @@ int check_privilege_from_ranger(List *request_list, List *result_list)
 	json_object_put(jrequest);
 
 	/* parse the JSON-format result */
-	RangerACLResult ret = parse_ranger_response(curl_context_ranger.response.buffer);
+	int ret = parse_ranger_response(curl_context_ranger.response.buffer, result_list);
+	if (ret < 0)
+	{
+		elog(WARNING, "parse ranger response failed, response[%s]", 
```

Comment:
The response content is a json format. like {""requestId"":1,""privileges"":[""usage""],""allowed"":true}]},  if we add "[]" outside it,  square brackets can be a part of json content. Do you think so  ?